isb_web/manage.py

```diff
@@ -154,12 +153,12 @@ async def __call__(
 )
 
 
-class MintIdentifierParams(BaseModel):
```

Comment:
Rename all the previous implementations to reflect the fact that they're DataCite specific.